todoapp/todos/models.py

```
@@ -16,3 +14,20 @@ class Todo(models.Model):
     """
+
+    name = models.CharField(max_length=1000, blank=False)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+    done = models.BooleanField(default=False)
+    date_created = models.DateTimeField(auto_now_add=True)
+    date_completed = models.DateTimeField(null=True, blank=True, editable=False)
+
+    def __str__(self):
+        return f"Todo Name: {self.name} by {self.user.email}"
+    
+    def save(self, *args, **kwargs):
```

Ruff (0.12.2)

Missing return type annotation for public function `save`

(ANN201)